src/index.html

```diff
@@ -1,12 +1,394 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400;500;600&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#logo" class="header__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+            Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#phone" class="nav__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#noutbuk"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadget" class="nav__link">
+              Гаджеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablet" class="nav__link">
+              Планшеты
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#foto" class="nav__link">
+              ФОТО
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              ВИДЕО
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main-page">
+      <section class="main-page__catalog catalog">
+        <article class="card catalog__card" data-qa="card">
+          <img
+            class="card__foto"
+            src="images/imac.jpeg"
+            alt="foot product">
+          <h2 class="card__h2">
```

Comment:
better use `title` class name
```suggestion
          <h2 class="card__title">
```

Reply:
змінила